cmds/webboot/network.go

```diff
@@ -157,6 +167,8 @@ func enterCredentials(uiEvents <-chan ui.Event, authSuite wifi.SecProto) ([]stri
 	identity, err := menu.PromptTextInput("Enter identity:", menu.AlwaysValid, uiEvents)
 	if err != nil {
 		return nil, err
+	} else if identity == "<Esc>" {
```

Comment:
I wonder if it'd make sense to create a new error type for back option and then switch on the error instead of having to check for esc? I think then you'd also be able to avoid adding the extra bool to the return type. Not sure, what do you think?

Reply:
Yeah I really like that idea.

Actually, that would also help make the return type for `PromptTextInput()` more consistent with the one for `PromptMenuEntry()`.

I'll make that change here, and I might have to add another PR to make some changes to `PromptMenuEntry()`.

Reply:
Yeah I should have designed a special error for <Escape> at the very beginning  instead of using the return content to recognize the <Escape> key. Just for reminding, not only the PromptTextInput() here but also the parsingMenuOption() is using the return content to recognize the <Escape> key and you may want to change all of them. Thank you for fix that!!